dev/bots/suite_runners/run_web_tests.dart

```diff
@@ -732,7 +732,7 @@ class WebTestsSuite {
         // and it doesn't use most of startCommand's features; we could simplify this a lot by
         // inlining the relevant parts of startCommand here.
         'chromedriver',
-        <String>['--port=4444'],
+        <String>['--port=4444', '--log-level=INFO', '--enable-chrome-logs'],
```

Comment:
I can't find any documentation online about `log-level` (?)

Reply:
I got it from the source:

```
$ chromedriver --help
Usage: chromedriver [OPTIONS]

Options
  --port=PORT                     port to listen on
  --adb-port=PORT                 adb server port
  --log-path=FILE                 write server log to file instead of stderr, increases log level to INFO
  --log-level=LEVEL               set log level: ALL, DEBUG, INFO, WARNING, SEVERE, OFF
  --verbose                       log verbosely (equivalent to --log-level=ALL)
  --silent                        log nothing (equivalent to --log-level=OFF)
  --append-log                    append log file instead of rewriting
  --replayable                    (experimental) log verbosely and don't truncate long strings so that the log can be replayed.
  --version                       print the version number and exit
  --url-base                      base URL path prefix for commands, e.g. wd/url
  --readable-timestamp            add readable timestamps to log
  --enable-chrome-logs            show logs from the browser (overrides other logging options)
  --bidi-mapper-path              custom bidi mapper path
  --allowed-ips=LIST              comma-separated allowlist of remote IP addresses which are allowed to connect to ChromeDriver
  --allowed-origins=LIST          comma-separated allowlist of request origins which are allowed to connect to ChromeDriver. Using `*` to allow any host origin is dangerous!
```

Reply:
The CLI never lies! :P